### DISCUSSION TITLE
I cant give gifts to monika

### DISCUSSION THREAD

**Discussion by KatsuroKazuki** [Monika-After-Story/MonikaModDev#9163] [QA - tech support + general] [answered]
I can only give her some ribbons, coffee, thermos etc. but whenever I try to give her new clothes she can't read the file.  I USe the spritpacks

**Comment by Yazijustmonika** [comment]
have you put the mod assets in the right place? and reminder if you wanna give her clothes the you need to copy paste the assets folder not when you inside the assets folder
![Capture8](https://user-images.githubusercontent.com/101006938/157608949-2ce48bb2-6605-493c-9abe-d83cc27176ac.JPG)

**Comment by KatsuroKazuki** [comment]
> have you put the mod assets in the right place? and reminder if you wanna give her clothes the you need to copy paste the assets folder not when you inside the assets folder ![Capture8](https://user-images.githubusercontent.com/101006938/157608949-2ce48bb2-6605-493c-9abe-d83cc27176ac.JPG)

If I Copy the assets folder where would I paste it? And Yes, I Copy the mod assets and paste inside "game" I have copy and paste the mod assets for multiple times already

**Comment by ThePotatoGuy** [comment, marked as answer]
follow the [troubleshooting steps here](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation)
